src/platform/silabs/ConnectivityManagerImpl_WIFI.cpp

```diff
@@ -426,6 +427,30 @@ void ConnectivityManagerImpl::UpdateInternetConnectivityState(void)
         event.InternetConnectivityChange.IPv6      = GetConnectivityChange(hadIPv6Conn, haveIPv6Conn);
         event.InternetConnectivityChange.ipAddress = addr;
 
+#if CHIP_CONFIG_ENABLE_ICD_SERVER
+        {
+            sl_wfx_mac_address_t macaddr;
+            wfx_get_wifi_mac_addr(SL_WFX_STA_INTERFACE, &macaddr);
+            char macaddrString[13]; // 12 characters + null terminator
+            sprintf(macaddrString, "%02X%02X%02X%02X%02X%02X", macaddr.octet[0], macaddr.octet[1], macaddr.octet[2],
+                    macaddr.octet[3], macaddr.octet[4], macaddr.octet[5]);
+            EndpointQueueFilterConfig config;
+            config.allowedQueuedPackets = 20; // Set the desired value
+
+            mEndpointQueueFilter.SetConfig(config);
+            if (mEndpointQueueFilter.SetHostName(chip::CharSpan(macaddrString)) == CHIP_NO_ERROR)
+            {
+                chip::Inet::UDPEndPointImpl::SetQueueFilter(&mEndpointQueueFilter);
```

Comment:
It's a little confusing why some things like `EndpointQueueFilterConfig` don't get a chip:: prefix while other things do....  None of them need it, given the `using namespace` above.

Reply:
done